### PR TITLE
added community landing page link to community sidebar

### PR DESCRIPTION
**Pull request by softcoder32**
When going to view community profile, there is no easy way to get back to community landing page so this PR addresses this problem.

### CODE REVIEW

**Comment by softcoder32** [comment]
community_profile

File: systers_portal/templates/community/snippets/community_sidebar.html [hunk 1]
```diff
@@ -5,6 +5,7 @@
     <div class="sidebar-module mb40">
       <h4>Community Actions</h4>
       <ol class="list-unstyled">
+        <li><a href="{% url 'view_community_landing' community.slug %}">Community Landing page</a></li>
```
Comment (tapasweni-pathak) [inline systers_portal/templates/community/snippets/community_sidebar.html]: Community Landing Page.

File: systers_portal/templates/community/view_profile.html [hunk 2]
```diff
@@ -12,34 +12,38 @@
...
-                  {% else %}
-                    <td>{{ value|urlize }}</td>
-                  {% endif %}
-                </tr>
-              {% endif %}
-            {% endwith %}
-          {% endfor %}
-          </tbody>
-        </table>
+      <div class="col-md-6">
```
Comment (tapasweni-pathak) [inline systers_portal/templates/community/view_profile.html]: ts or space? editor issues, I guess.

File: systers_portal/templates/community/view_profile.html [hunk 3]
```diff
@@ -12,34 +12,38 @@
...
+                  </tr>
+                {% endif %}
+              {% endwith %}
+            {% endfor %}
+            </tbody>
+          </table>
+        </div>
       </div>
-    </div>
+      <div class="col-md-3 col-md-offset-3 sidebar-module mb40">
```
Comment (tapasweni-pathak) [inline systers_portal/templates/community/view_profile.html]: :100:

**Pull request merged**